### DISCUSSION TITLE
Match probability

### DISCUSSION THREAD

**Discussion by msiemionCalistapw** [moj-analytical-services/splink#733] [Q&A] [answered]
How is the match probability and match weight calculated  for each record pair? I know this is more involved as there are blocking rules and comparison setting but is there a general mathematical approach?

**Comment by RobinL** [comment, marked as answer]
See the following two articles on my website:
[Introduction to probabilistic linkage](https://www.robinlinacre.com/intro_to_probabilistic_linkage/)
[The mathematics of the Fellegi Sunter model](https://www.robinlinacre.com/maths_of_fellegi_sunter/)

**Comment by austinweisgrau** [comment]
Followup question here - should match probability be taken as a complete calculation of the probability that two records are a match, or should match weight be factored in as well? For example, would it be appropriate to consider match probability without considering match weight, or should the two numbers be multiplied or something to generate a useable figure?

**Reply by RossKen** [reply]
Only one needs to be considered. Probability is derived from Match Weight - if you look at the [waterfall chart ](https://moj-analytical-services.github.io/splink/demos/06_Visualising_predictions.html#waterfall-chart), you can see how they compare with match weight on the left axis and probability on the right.
